src/flow_dot_js.ml

```diff
@@ -168,6 +168,63 @@ let check_content_js js_file js_content =
   let content = Js.to_string js_content in
   check_content ~filename ~content
 
+let mk_loc file line col =
+  {
+    Loc.
+    source = Some file;
+    start = { Loc.line; column = col; offset = 0; };
+    _end = { Loc.line; column = col + 1; offset = 0; };
+  }
+
+let infer_type filename content line col =
+    let filename = Loc.SourceFile filename in
+    let root = Path.dummy_path in
+    (* let verbose = Some(0) in *)
```

Comment:
Sure, thanks for pointing out.
On Thu, 2 Jun 2016 at 22:35, Marshall Roch <notifications@github.com> wrote:

> In src/flow_dot_js.ml
> <https://github.com/facebook/flow/pull/1889#discussion_r65604121>:
>
> > @@ -168,6 +168,63 @@ let check_content_js js_file js_content =
> >    let content = Js.to_string js_content in
> >    check_content ~filename ~content
> >
> > +let mk_loc file line col =
> > +  {
> > +    Loc.
> > +    source = Some file;
> > +    start = { Loc.line; column = col; offset = 0; };
> > +    _end = { Loc.line; column = col + 1; offset = 0; };
> > +  }
> > +
> > +let infer_type filename content line col =
> > +    let filename = Loc.SourceFile filename in
> > +    let root = Path.dummy_path in
> > +    (* let verbose = Some(0) in *)
>
> can you delete this stuff?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/flow/pull/1889/files/8bf90d679f22a1e82935424e2552826acb4f8eee#r65604121>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA9SU2OXKGEI6rp5NjrH5eCqhPSd3Lmzks5qHzCOgaJpZM4Is4d->
> .
>